[HDRP Sample Scene] Tutorial updates for the Probe Volumes sections

pierre-unity opened the pull request:
### Purpose of this PR
This PR intends to polish the tutorial sections about the Probe Volumes.

Changes

**Page 1:**
- Added mention of "Adaptive Probe Volumes" and "APV" (the whole tutorial only use "Probe Volumes" to follow internal guidelines from Doc and Design)
- Added mention of "Indirect **diffuse** lighting"
- Added mention of the higher density of probes around geometries, and lower density in empty spaces
- Added mention of faster workflow compared to previous light probes
- Added mention of better quality when probe volume density is higher than lightmap resolution
- Added precision about the benefits of the reflection probe normalization (specular leaking reduction)


**Page 2:**
- New page dedicated only to toggling and debugging probes

**Page 3:**
- Added mention of light leaking in the intro paragraph
- Added last step to revert to previous state

**Page 4:** 
- Clearer differentiation between the "Global" virtual offset, and then "local" virtual offset via the Adjustment Volumes
- Added mention of "Additional Properties..." as the property might not be visible to the user

**Page 5:**
- Removal of the "might be difficult to spot..." and reframed the first few paragraphs
- Added a last step to revert to the previous sampling noise settings

**Page 6:**
- Rephrasing of how the local Probe Volume can force a fixed distance between probes

**Page 7:**
- Added more precision about the scenarios that will require the use of layer masks (distant objects, proxies)
- Explanation about the use case for the ShadowProxy and why it was created in the first place
- Added steps to ask the user to ensure they have the probe debug view enabled
- Added a step to explain what the user is actually seeing (no probes in empty space within the ShadowProxy)

**Page 8:**
- Very minor tweaks

---
### Testing status
Describe what manual/automated tests were performed for this PR

---
### Comments to reviewers
Risks are very minimal as these changes are only textual.

MaximeGrangeUnity commented on com.unity.template-hd/Assets/SampleSceneAssets/Tutorials/Pages/LightProbes_03.asset:
```diff
@@ -39,41 +39,26 @@ MonoBehaviour:
-          <b>Generate Lighting.</b>'
+        m_Untranslated: "Because Probe Volumes only offer automatic probe placement,
+          individual probes cannot be moved or deleted. This is a core limitation
+          of the system. \n\nIn case you need to move a set of probes, you can instead
```
```suggestion
          of the system. \n\nIn case you need to move a set of probes, you can 
```

MaximeGrangeUnity replied:
two 'instead' in the same sentence

MaximeGrangeUnity commented on com.unity.template-hd/Assets/SampleSceneAssets/Tutorials/Pages/LightProbes_05.asset:
```diff
@@ -39,19 +39,23 @@ MonoBehaviour:
+          these steps:\n\n1. Add a Cube at <b>X:25.3 Y:2 Z:1.5</b>\n\n2. In the <i>Hierarchy</i>
+          window select <b>Lighting</b> > <b>Probe Volumes</b> > <b>Probe Volume
+          Local.</b>\n\nNotice that the range slider has its Min and Max value set
+          at 0.6m.\n\n3. Disable the <b>Probe Volume Local</b> GameObject and save
```
```suggestion
          at 0.6m, and that the _Fill Empty Space_ option is enabled.\n\n3. Disable the <b>Probe Volume Local</b> GameObject and save
```

MaximeGrangeUnity commented:
Nice rephrasing and addition to the tutorial. Just noticed a syntax error, and a step missing in the instructions for the Local Probe Volumes section  (see above comments).

iM0ve commented:
Generally a welcome change for the tutorial. In some specific cases a bit of information was lost in translation. But its subjective so will post it for consideration and not blocking request.

1). In Virtual offset. The previous version I think more clearly described that there are 2 options for VO - one is automatic the other is manual. In the new one this can be read between the lines, but I think old was clearer.

OLD
![image](https://github.com/Unity-Technologies/Graphics/assets/36502659/f7701a06-c8df-4bde-867c-fc449e38921a)

NEW
![image](https://github.com/Unity-Technologies/Graphics/assets/36502659/4faa4484-d020-4e8d-b17d-f1345a47459b)

2. Probe Volumes. The goal of this section was to introduce the term Subdivision level (as it causes the issue highlighted in the step). This was done by request from Maxime. In the new version we dont mention the term and in the next paragraph we use it expecting user to know. Also I like the more simplistic writing style but this is highly subjective so I dont mind.

OLD
![image](https://github.com/Unity-Technologies/Graphics/assets/36502659/6252d650-1d96-4cf0-bc7c-cb71fa5d9a92)

NEW
![image](https://github.com/Unity-Technologies/Graphics/assets/36502659/431adbfb-0539-46c3-b6bc-eb45bc5bf2da)

Also in the same step we had a small description of what the issue is, because its a bit minor and hard to spot. In the new version there is no longer any description so it might not be clear what kind of artifact we have in mind.
![image](https://github.com/Unity-Technologies/Graphics/assets/36502659/9a6e5db7-5204-4b17-ba3a-5b9cdfcd061b)

One last note about the section. This is very minor but in previous version we recommended to set sampling noise to 0.1 (default value) because likely when users first bake most of their settings will be default. Then they will bump into issues and look for solutions. So its more realistic to use 0.1 as well as I can see from the tutorial image, that the sampling is not set as 0.

pierre-unity commented:
Very good feedback, thank you!

1.  I've updated the text for the Virtual Offset to integrate your feedback:

> However, if certain probes are in unwanted locations (e.g. inside or behind geometries), you can let the system alter the capture point of all probes in the Scene, by tuning the global <b>Virtual Offset</b> properties in the <i>Lighting</i> window, under the <b>Probe Volumes</b> tab. 

For finer localized probe adjustments in the Scene, you may use instead a dedicated <b>Probe Adjustment Volume</b> found under <b>GameObject</b> > <b>Light</b> > <b>Probe Volumes.</b>"


2. The very first paragraph did contains "subdivision level" already, but I've now added it in bold:

> We'll now cover indirect lighting sampling issues between probes having different spacing distances, or <b>Subdivision Levels</b>.

What's complicated is that 3 distances (0.6, 1.8 and 5.4 in this example) means that there are only 2 subdivision levels (N minus 1, in practice), so it's a bit complicated to explain. That's why I'd rather brush over it, given that the prevalence of "Subdivision levels" in the UI is very minimal, we use the spacing distances instead.

So, I've reformulated the 2nd paragraph this way, to integrate your feedback:

> In this particular Scene, the spacing between Probes will fluctuate between different <b>Subdivision Levels</b> at 0.6m, 1.8m and 5.4m. 

Regarding the image being unclear, I have already updated it in the PR from Arttu (the seam is very visible, it's a harsh line now). This is why I removed the mention of the artifact being "difficult to spot". The image was captured with a sampling noise value of 0. I think this is fine to keep it this way, because it's a real-time effect which doesn't require a rebake, so they can play with whatever value they please.

3. Agreed, I've now reformulated the first paragraph now (I still found the "Old" version to be a bit unclear). ;) 

>In certain situations, the sampling between serveral <b>Subdivision Levels</b> can produce visible lighting seams, as seen in the image above.

>Thankfully, by adding a Local <b> Probe Volume</b> in the Scene and enabling its <b>Fill Empty Spaces</b> property, we can enforce an uniform spacing between all probes.

4. Yes, thank you for the reminder, I did it to have easier comparison between file names (for GitHub). Once you're happy with the corrections above, I'll rename everything from _01 onwards. ;)

pierre-unity commented:
All the feedback from Motiejus and Maxime has been implemented. I have refocused the titles for 2 pages, however "Sampling Noise", and "Uniform Spacing" pages.

pierre-unity commented:
Final tweak as Motiejus requested: The file naming has been updated (and I'm now using "ProbeVolumes..." instead of "LightProbes..." for file names

![image](https://github.com/Unity-Technologies/Graphics/assets/39901544/e3b051c7-b719-4af5-aa2c-c4e74f42f239)

I've also normalized the settings for all image files, as it was quite random. This also fixes some interpolation errors on certain images of the tutorials (bright lights at the bottom of certain images)
![image](https://github.com/Unity-Technologies/Graphics/assets/39901544/e5afc03c-e637-474b-9711-abd3b16febb9)

sebastienlagarde reviewed (approved):
some files to remove before merging, I poked Kristin about it.
